discord/ext/commands/cog.py

```diff
@@ -49,12 +49,12 @@ def __new__(cls: type[CogT], *args: Any, **kwargs: Any) -> CogT:
         # To do this, we need to interfere with the Cog creation process.
         return super().__new__(cls)
 
-    def walk_commands(self) -> Generator[Command, None, None]:
+    def walk_commands(self) -> Generator[Invokable, None, None]:
         """An iterator that recursively walks through this cog's commands and subcommands.
 
         Yields
         ------
-        Union[:class:`.Command`, :class:`.Group`]
+        Union[:class:`.Invokable`]
```

Comment:
```suggestion
        :class:`.Invokable`
```
A union with only one member is invalid (according to static type checkers).